docs/JSON-RPC-API-using-jmwalletd.md

```diff
@@ -0,0 +1,284 @@
+## JSON-RPC API for Joinmarket using jmwalletd.py
+
+### Introduction - how to start the server
+
+Create an ssl certificate and store it in `scripts/ssl/{key,cert}.pem`.
```

Comment:
Oops, that's a screw up thanks. Will fix.

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/996/commits/37ecee6385c6a0ee4729b340e96256346a745b90, except made it more sane: it's now in `datadir/ssl/{key,cert}.pem` where your `datadir` is defined by `jm_single().datadir` with the same rules as elsewhere in the application (note that in regtest testing it's set to "." instead of `~/.joinmarket`). Doc is updated for that.